### PR TITLE
docs: replace HCL code with correct language specific code

### PR DESCRIPTION
**Pull request by tmeckel**
Removed left over HCL code with Pulumi specific code

Related: https://github.com/pulumi/registry/pull/4184

### CODE REVIEW

File: docs/installation-configuration.md [hunk 1]
```diff
@@ -1293,52 +1293,157 @@ If the port is changed or intended to be changed, refer to the details below
...
-  admin_sport    = 8443
-  admin_ssh_port = 22
-}
-```
+    ```yaml
+    provider "fortios" {
+      hostname = "192.168.52.111"
+      token    = "nx6nbGn8tnFddaa3Qy79jpjfsyw1"
+      insecure = "true"
+    }
```
Comment (cnunciato) [inline docs/installation-configuration.md]: This doesn't look right.

File: docs/installation-configuration.md [hunk 2]
```diff
@@ -1293,52 +1293,157 @@ If the port is changed or intended to be changed, refer to the details below
...
+2. Change the Provider part of the configuration file `Pulumi.yaml`, add the
+   8443 port number to the hostname and separate them with a colon, as follows,
+   then re-apply Pulumi:
+
+    ```yaml
+    provider "fortios" {
+      hostname = "192.168.52.111:8443"
+      token    = "nx6nbGn8tnFddaa3Qy79jpjfsyw1"
+      insecure = "true"
+    }
```
Comment (cnunciato) [inline docs/installation-configuration.md]: This doesn't look right either. Apologies if I'm missing something though -- I admit I'm completely unfamiliar with this provider, but this looks like HCL to me.

File: docs/installation-configuration.md [hunk 3]
```diff
@@ -1293,52 +1293,157 @@ If the port is changed or intended to be changed, refer to the details below
...
+    {{< chooser language "typescript,python,go,csharp,yaml" >}}
 
-```hcl
-provider "fortios" {
-  hostname = "192.168.52.111:8443"
-  token    = "nx6nbGn8tnFddaa3Qy79jpjfsyw1"
-  insecure = "true"
-}
-```
+    {{% choosable language typescript %}}}
```
Comment (cnunciato) [inline docs/installation-configuration.md]: I think this extra brace may be messing with the formatting:

```suggestion
    {{% choosable language typescript %}}
```

Comment (tmeckel) [reply docs/installation-configuration.md]: @cnunciato typical copy-paste error. Found a couple of these. Thanks for finding!

File: docs/installation-configuration.md (hunk 1 above)
Comment (tmeckel) [reply docs/installation-configuration.md]: fixed

**Comment by tmeckel** [comment]
link: https://github.com/pulumi/registry/pull/4184

**Pull request merged**